ONSdigital/ras-frontstage
# Resolve test deprecations

# What and why?
This PR addresses the deprecation warnings seen when running the tests locally. 

Setuptools uses a deprecated call to pkg_resources which isn't managed by us. After updating setuptools to the most recent version and no change to the deprecation warnings, this part has been split out into a separate card to investigate a new approach. 

Pytest doesn't like non-test classes starting with Test so this has been renamed. 
# How to test?
run the tests locally, should no longer see a warning message for PytestCollectionWarning.
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&view=detail&selectedIssue=RAS-925
